addons/tasks/XEH_preInit.sqf

```diff
@@ -6,14 +6,17 @@ PREP_RECOMPILE_START;
 PREP_RECOMPILE_END;
 
 GVAR(tasks) = call CBA_fnc_createNamespace;
+GVAR(tasksArray) = [];
```

Comment:
@3Mydlo3 are these needed on clients? I think it should be moved under `isServer` check.

Reply:
I think it's not needed on clients at all.